src/pystatis/http_helper.py

```diff
@@ -92,7 +92,18 @@ def get_data_from_endpoint(
     Returns:
         requests.Response: the response object holding the response from calling the Destatis endpoint.
     """
-    db_host, db_user, db_pw = db.get_db_settings()
+    
+    # Determine database by matching regex to item code
+    name = params.get("name", params.get("selection", ""))
```

Comment:
this can also fail for endpoints like /catalogue/cubes2statistics that accept partial names or /vatalogue/values that has a selection param but that is TODUR1 for example so no EVAS at all. We have to rethink this approach

Reply:
should be done...either the method gets a db_name or we try to find it in params. If both fails, the method fails